readme.md

```diff
@@ -12,11 +12,15 @@
 
 **Orleans.Http** is a package that use ASP.Net Core 3.0 as a frontend endpoint for Orleans grains without the need of controller classes.
 
-> **NOTE**: This is a work-in-progress.
+This package leverages AspNet Core 3.0 Endpoint Routing to expose grains as Http endpoints without need of have to implement Controllers. The request is received and processed by grains methods itself. The idea is to perform in a similar way to Controllers without having to add any boilerplate code to your Orleans project.
```

Comment:
s/AspNet/ASP.NET

Maybe drop "3.0" since that will become quickly out-dated

s/Http/HTTP